src/vega.schema.ts

```diff
@@ -1,17 +1,17 @@
 import {
   AggregateOp,
-  Align,
+  Align as VgAlign,
```

Comment:
Are these all different in Vega? (If they are the same, we don't need to add Vg prefix?)

Reply:
No, they are the same. I will change this then.